src/fs/src/lib.rs

```diff
@@ -35,7 +35,8 @@ impl Metadata {
         // Create metadata file.
         let mut file = std::fs::OpenOptions::new();
 
-        file.create_new(true);
+        file.create(true);
+        file.truncate(true);
```

Comment:
The reason we don't use truncate here because we want to catch if the game provides the OBM files.